### DISCUSSION TITLE
When combining jpg to pdf, the default size from get_pixmap is 3000+, but when I print with acrobat to an A4 pdf, the  default size from get_pixmap is  960 x 541 pixels, etc.? How to set all pages to a normal size?

### DISCUSSION THREAD

**Discussion by nissansz** [pymupdf/PyMuPDF#4077] [Q&A]
When combining jpg to pdf, the default size from get_pixmap is 3000+, but when I print with acrobat to an A4 pdf, the  default size from get_pixmap is  960 x 541 pixels, etc.? How to set all pages to a normal size for all pdf files?

**Comment by JorjMcKie** [comment]
> When combining jpg to pdf, ...

How do you do that "combining"?

Otherwise: the pixmap dimensions (i.e. its rectangle) reflect the number of pixels in x / y direction. This is not the same as the page size when printed / displayed. Here, the resolution comes into play.

**Reply by nissansz** [reply]
I use acrobat to combine

**Reply by JorjMcKie** [reply]
Then the Acrobat documentation should answer that question.
